src/applications/disability-benefits/686/config/form.js

```diff
@@ -127,23 +392,23 @@ const formConfig = {
   footerContent: FormFooter,
   getHelp: GetFormHelp,
   defaultDefinitions: {
-    address,
-    marriages,
+    location,
+    postalCode,
+    previousMarriages,
     fullName,
     ssn,
     date,
     vaFileNumber,
   },
   chapters: {
-    applicantInformation: {
-      title: 'Applicant Information',
+    veteranInformation: {
+      title: 'Veteran Information',
```

Comment:
Although we are now calling this property veteran information, it may still make sense to call this section "applicant information", or even "claimant information".
If applicants are only able to submit applications on their own behalf, we tend to use "claimant", if it is possible for an applicant to apply on behalf of another person, we will also provide "applicant" as an option. In the event that an "applicant" is filling out a form on behalf of a veteran, we will title the veteran-specific sections "veteran" information. As only the veteran is now able to apply, we may choose to use "claimant" here. Could you please add a TODO comment here for us to verify this content?